portfolio/src/main/java/com/google/sps/Puzzle.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps;
+
+/** Puzzle domain object */
+public class Puzzle {
+
+  private String puzzleId; //Datastore generated id
```

Comment:
Here and below:
Java style dictates that Javadocs should go right above the variable declaration.
```suggestion
  /**
   * Datastore generated id
   */
  private String puzzleId; 
```